src/parse/removeDecorators.ts

```diff
@@ -11,7 +11,7 @@ import backtrace from '../error/backtrace.js';
 import logger from '../utils/logger.js';
 
 // regex: matches all cases of 'known' unless they are directly preceded by 'un'
```

Comment:
This all works, could you please update the regex comment here? Something like `regex: matches decorators when standalone words`, thanks :)

Reply:
Done . Thanks